CauldronMods/Controller/Heroes/LadyOfTheWood/Cards/FallCardController.cs

```diff
@@ -14,31 +14,26 @@ public override void AddTriggers()
 		{
 			//Whenever LadyOfTheWood deals lightning damage to a target, reduce damage dealt by that target by 1 until the start of your next turn.
 			Func<DealDamageAction,bool> criteria = (DealDamageAction dd) => dd.DamageSource != null && dd.DamageSource.IsSameCard(base.CharacterCard) && dd.DamageType == DamageType.Lightning;
-			base.AddTrigger<DealDamageAction>(criteria, new Func<DealDamageAction, IEnumerator>(this.ReduceDamageResponse), new TriggerType[]
+			base.AddTrigger<DealDamageAction>(criteria,
+				(DealDamageAction dd) => AddStatusEffectResponseToDamage(dd, (DealDamageAction damage) =>
+					ReduceDamageDealtByThatTargetUntilTheStartOfYourNextTurnResponse(damage, 1), requireDamageToBeDealt: true
+				), new TriggerType[]
 			{
 				TriggerType.ReduceDamage
-			}, TriggerTiming.After);
+			}, TriggerTiming.Before);
 		}
-
-		private IEnumerator ReduceDamageResponse(DealDamageAction dd)
+		protected IEnumerator AddStatusEffectResponseToDamage(DealDamageAction dd, Func<DealDamageAction, IEnumerator> statusEffectResponse, bool requireDamageToBeDealt = false)
```

Comment:
Since this is a protected method, and it's only called once with "requireDamageToBeDealt" as true, this method could be simplified by removing it, but I think it makes the method more clearly match the text on the card (plus, I just copied it from Nitrogen Rounds, and it has the same redundancy).